R/helpers.R

```diff
@@ -799,18 +799,36 @@ cell_style_to_html.cell_text <- function(style) {
 #'
 #' @param color The fill color. If nothing is provided, then `"#D3D3D3"` (light
 #'   gray) will be used as a default.
+#' @param alpha The alpha transparency value for the `color` as single value in
+#'   the range of `0` (fully transparent) to `1` (fully opaque). If not provided
+#'   the fill color will be fully opaque.
 #'
 #' @family helper functions
 #' @export
-cell_fill <- function(color = NULL) {
+cell_fill <- function(color = NULL,
```

Comment:
set defaults to `#D3D3D3` and `1` like we did for another helper.

Reply:
Done. Thanks!